GUI/Dialogs/ManageGameInstancesDialog.cs

```diff
@@ -73,42 +74,66 @@ public void UpdateInstancesList()
             GameInstancesListView.Items.Clear();
             UpdateButtonState();
 
+
+            if (!GameInstancesListView.Columns.Contains(Game))
+            {
+                // Always show the game column so our rows load correctly
+                GameInstancesListView.Columns.Insert(Game.Index, Game);
+            }
             if (!GameInstancesListView.Columns.Contains(GamePlayTime))
             {
```

Comment:
Hmm, the add-then-remove logic of this was based on the belief that the _contents_ of the columns were hidden when the columns were hidden (and I thought I confirmed that to be the case in testing, but I may be mis-remembering). If that's not the case, then we can probably simplify this to add/remove the columns all together and then just add the column data we need when we add the rows, as you have it now in `rowItems`. In other words, we don't need to call `Insert` for either of these columns if we aren't planning to populate them.

Reply:
Added a commit based on this idea, does it look OK to you?

Reply:
> based on the belief that the _contents_ of the columns were hidden when the columns were hidden (and I thought I confirmed that to be the case in testing, but I may be mis-remembering)

I can imagine that this is true on Windows, but broken on Mono (assuming you also were on Windows at that time).

> Added a commit based on this idea, does it look OK to you?

Very nice, makes it cleaner.